Gems/GraphCanvas/Code/StaticLib/GraphCanvas/Editor/GraphModelBus.h

```diff
@@ -213,7 +213,16 @@ namespace GraphCanvas
         {
         }
         ////
-        
+
+        //////////////////////////////////////
+        // Breakpoint handling
+
+        virtual void AddBreakpoints([[maybe_unused]] const AZStd::unordered_set<NodeId>& nodeIds)
+        {
+        }
+
```

Comment:
Used to communicate between GraphCanvas UI and its implementation (in our case ScriptCanvas, but there is also MaterialEditor and so on)